server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -88,7 +183,7 @@ async fn list_messageattempts(
         permissions: _,
         app,
     }: AuthenticatedApplication,
-) -> Result<Json<ListResponse<MessageAttemptOut>>> {
+) -> Result<Json<ListResponse<MessageAttemptOut>>, Error> {
```

Comment:
Why not use the `Result` that is already defined this way like it was?

Reply:
Deriving `FromQueryResult` when the custom `Result` alias errors  because `this  type alias takes 1 generic argument, but 2 generic arguments  were supplied`.  I realizes this has been fixed in the latest release of SeaORM (by you nonetheless :smile: ), so we can fix this by bumping the version.

Reply:
Fix: 😎 

Though why is it needed all of a sudden in all of these places? Happy to also upgrade if needed.